README.md

```diff
@@ -75,13 +254,170 @@ This option aims to limit the number of operations made with the GitHub API to a
 Based on your project, your GitHub business plan and the date of the cron job you set for this action, you can increase this limit to a higher number.
 If you are not sure which is the right value for you or if the default value is good enough, you could enable the logs and look at the end of the stale action.  
 If you reached the limit, you will see a warning message in the logs, telling you that you should increase the number of operations.
-If you choose not to increase the limit, you might end up with un-processed issues or pull requests after a stale action run.
+If you choose not to increase the limit, you might end up with unprocessed issues or pull requests after a stale action run.
 
 When [debugging](#Debugging), you can set it to a much higher number like `1000` since there will be fewer operations made with the GitHub API.  
 Only the [actor](#repo-token) and the batch of issues (100 per batch) will consume the operations.
 
 Default value: `30`
 
+#### remove-stale-when-updated
+
+Automatically remove the stale label when the issues or the pull requests are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`) or commented.
+
+Default value: `true`
+
+#### remove-issue-stale-when-updated
+
+Override [remove-stale-when-updated](#remove-stale-when-updated) but only to automatically remove the stale label when the issues are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`) or commented.
+
+Default value: unset
+
+#### remove-pr-stale-when-updated
+
+Override [remove-stale-when-updated](#remove-stale-when-updated) but only to automatically remove the stale label when the pull requests are updated (based on [GitHub issue](https://docs.github.com/en/rest/reference/issues) field `updated_at`) or commented.
+
+Default value: unset
+
+#### debug-only
+
+Run the stale workflow as dry-run.  
+No GitHub API calls that can alter your issues and pull requests will happen.  
+Useful to debug or when you want to configure the stale workflow safely.
+
+Default value: `false`
+
+#### ascending
+
+Change the order used to fetch the issues and pull requests from GitHub.
+`true` is for ascending.  
+`false` is for descending.
+
+It can be useful if your repository is processing so much issues and pull requests that you reach the [operations-per-run](#operations-per-run) limit.  
```

Comment:
```suggestion
It can be useful if your repository is processing so many issues and pull requests that you reach the [operations-per-run](#operations-per-run) limit.
```

Reply:
```suggestion
It can be useful if your repository is processing so many issues and pull requests that you reach the [operations-per-run](#operations-per-run) limit.  
```